extensions/wikia/NewWindowLinks/NewWindowLinks.php

```diff
@@ -64,7 +64,8 @@ function efParserCreateLink( $parser, $target, $label = null ) {
         $label = $parser->recursiveTagParse( $label );
         $label = preg_replace( "/\b({$parser->mUrlProtocols}) /", "$1", $label ); // Put them back together
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)